### PR TITLE
Upgrade to Angular 11

### PR DESCRIPTION
**Pull request by marko164**
`ng test` and `ng lint` both pass

I removed dependecy to `@angular/router` since `DevModule` has only one component. If this changes in the future, I think `@angular/router` should be added to dev dependencies.

### CODE REVIEW

**Comment by liwonder** [comment]
@rsaenen @marko164 Can we use the 'peerDependency' instead of 'dependency'?

**Comment by marko164** [comment]
I think peer dependency would require parent project to have it. And not every angular project must depend on @angular/router

File: package.json [hunk 1]
```diff
@@ -49,36 +49,35 @@
...
   "peerDependencies": {},
   "dependencies": {
-    "@angular/common": "~10.1.2",
-    "@angular/compiler": "~10.1.2",
-    "@angular/core": "~10.1.2",
-    "@angular/forms": "~10.1.2",
-    "@angular/platform-browser": "~10.1.2",
-    "@angular/platform-browser-dynamic": "~10.1.2",
-    "@angular/router": "~10.1.2",
+    "@angular/common": "~11.0.0",
```
Comment (rsaenen) [inline package.json:52]: Could you migrate core/common/forms to peerDependencies ?

**Review by rsaenen — CHANGES_REQUESTED** [review]

**Comment by rsaenen** [comment]
Thanks a lot to keep this package up!

**Comment by smnbbrv** [comment]
@rsaenen why does the lib get broken with every angular update? Can this issue be properly handled that it's not failing with next versions?

**Comment by rsaenen** [comment]
@smnbbrv This package has basic functions and follows guidelines.
I think build isn't retrocompatible with each Angular major releases, afaik this package needs `ng update` and keep `ng-packagr` updated

**Comment by rsaenen** [comment]
Released with 11.0.0

**Pull request merged**